### PR TITLE
Show details within tiles on the top

### LINKED ISSUE #1234: PoolDetail: Show details within tiles on the top
part of #1161

### PR DESCRIPTION
**Pull request by junkai121**
close #1234

### CODE REVIEW

File: src/renderer/i18n/en/deposit.ts [hunk 1]
```diff
@@ -35,6 +35,13 @@ const deposit: DepositMessages = {
   'deposit.poolDetails.allTimeVal': 'All time volume',
   'deposit.poolDetails.totalSwaps': 'Total swaps',
   'deposit.poolDetails.totalUsers': 'Total users',
+  'deposit.poolDetails.liquidity': 'Liquidity',
+  'deposit.poolDetails.volumn': 'Volumn',
+  'deposit.poolDetails.earnings': 'Earnings',
+  'deposit.poolDetails.fees': 'Fees',
+  'deposit.poolDetails.totalTx': 'Total Tx',
+  'deposit.poolDetails.members': 'Members',
+  'deposit.poolDetails.apy': 'APY',
```
Comment (veado) [inline src/renderer/i18n/en/deposit.ts:44]: It seems we have some some of translations at other places as well (e.g. `Liquidity`, `Earnings`, `Fees` etc.). If so, pls extract duplications to `common.xyz`

**Review by veado — CHANGES_REQUESTED** [review]

File: src/renderer/services/midgard/pools.ts [hunk 2]
```diff
@@ -558,31 +560,35 @@ const createPoolsService = (
...
-          O.fold(
-            () => Rx.of(RD.initial),
-            (asset) =>
-              apiGetPoolStatsDetail$({
-                asset: assetToString(asset),
-                period: period
-              })
-          )
+  const poolStatsDetail$: PoolStatsDetailLD = Rx.combineLatest([selectedPoolAsset$, reloadSelectedPoolDetail$]).pipe(
+    RxOp.switchMap(([oSelectedPoolAsset, delay]) =>
```
[the remark this answers is not in the document]
Comment (thatStrangeGuyThorchain) [reply src/renderer/services/midgard/pools.ts:564]: actually.... you are the person who had added it 😅 delay needed for Deposit views to reload pool's data after any Depoist/withdraw actions after 5 secs

[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/services/midgard/pools.ts:564]: If I remember correctly, in _Deposit_ we do need it for re-loading shares after sending deposit tx. However, we don't need it here and can just ignore handling of `delay`.

[the remark this answers is not in the document]
Comment (thatStrangeGuyThorchain) [reply src/renderer/services/midgard/pools.ts:564]: 👌

**Review by veado — DISMISSED** [review]

**Pull request merged**